runtime/src/non_circulating_supply.rs

```diff
@@ -41,7 +41,10 @@ pub fn calculate_non_circulating_supply(bank: &Arc<Bank>) -> NonCirculatingSuppl
         )
     } else {
         bank.get_program_accounts(&solana_stake_program::id())
-    };
+    }
+    // TODO: Figure out how to properly handle errors here
+    .unwrap_or_default();
```

Comment:
@CriesofCarrots not 100% sure the best way to handle this yet, specifically where this method is called from `process_rest()` in `impl RequestMiddleware for RpcRequestMiddleware`, is there a best way to plumb up an error?

Reply:
Oh looks like that path in `process_rest()`  always uses a `root_bank`, and only unrooted forks can be removed, so maybe it's safe to just unwrap there

Reply:
Yeah, looks right to me.